java/adapter/orc/pom.xml

```diff
@@ -52,10 +52,27 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client-runtime</artifactId>
+            <version>3.3.2</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
```

Comment:
Can we add a brief comment about why we need to exclude these?

Reply:
Yes, the reason is `[WARNING] Rule 0: org.apache.maven.plugins.enforcer.BannedDependencies failed with message:
Found Banned Dependency: commons-logging:commons-logging:jar:1.1.3`. Let me add some message on the exclude tag.

Reply:
Ah, I didn't realize we had a bannedDependencies set. Ok, it's probably fine to leave this as is then.

Reply:
Though uh wow, that's been there since the initial Arrow commit in 2016. Are we sure Hadoop functions without this dependency, incidentally?

Reply:
Unit test is working without problems in JSE 8, JSE 11 but not in JSE 17, that is the reason because we added now.